internals/testintegration/pebble_run_test.go

```diff
@@ -0,0 +1,51 @@
+//go:build integration
+
+package testintegration_test
+
+import (
+	"fmt"
+	"os"
+	"testing"
+
+	. "github.com/canonical/pebble/internals/testintegration"
+)
+
+func TestMain(m *testing.M) {
+	if err := Setup(); err != nil {
+		fmt.Println("Setup failed with error:", err)
+		os.Exit(1)
+	}
+
+	exitVal := m.Run()
+	os.Exit(exitVal)
+}
+
+func TestPebbleRunWithSimpleLayer(t *testing.T) {
+	pebbleDir := t.TempDir()
+
+	layerYAML := `
+services:
+    demo-service:
+        override: replace
+        command: sleep 1000
+        startup: enabled
+    demo-service2:
+        override: replace
+        command: sleep 1000
+        startup: enabled
+`[1:]
```

Comment:
Yaml is actually pretty chill wrt. initial indent.
This should work just as well:

```suggestion
	layerYAML := `---
        services:
            demo-service:
                override: replace
                command: sleep 1000
                startup: enabled
            demo-service2:
                override: replace
                command: sleep 1000
                startup: enabled`

```